conf/st2.conf.sample

```diff
@@ -279,6 +279,8 @@ pool_size = 10
 retry_wait_msec = 3000
 # How often (in seconds) to look for zombie execution requests before rescheduling them.
 gc_interval = 10
+# How often in minutes to check for unscheduled actions
```

Comment:
```suggestion
# How long GC to search back in minutes for orphaned scheduled actions
```
The comment/description should match help in `config.py`.

I'm also unsure why Travis job is not triggering for your PR. Can you try to avoid force-pushing?